yal/assessment_data/connectedness.py

```diff
@@ -0,0 +1,27 @@
+# Assessment state naming convention
+# state_a[number]_[subsection]_q[question]_topic
+ASSESSMENT_QUESTIONS = {
+    "1":
+    # A4 Connectedness
+    {
+        "start": "state_a4_q1_connectedness",
+        "questions": {
+            "state_a4_q1_connectedness": {
+                "type": "button",
+                "text": "\n".join(
+                    [
+                        "[persona_emoji] *Do you have someone to talk to when you "
+                        "have a worry or problem?*",
+                    ]
+                ),
+                "options": [
+                    "Never",
+                    "Sometimes",
+                    "Most of the time",
+                    "All the time",
```

Comment:
We can only have 3 buttons per message, so I think this needs to be `list` type question (and the same for any other button questions with more than 3 options)